collectors/plugins.d/plugins_d.h

```diff
@@ -24,6 +24,25 @@
 #define PLUGINSD_KEYWORD_TOMBSTONE "TOMBSTONE"
 #define PLUGINSD_KEYWORD_HOST "HOST"
 
+//Replication keywords
+#define PLUGINSD_KEYWORD_REP "REP"
+#define PLUGINSD_KEYWORD_REP_ON "ON"
+#define PLUGINSD_KEYWORD_REP_OFF "OFF"
+#define PLUGINSD_KEYWORD_REP_ACK "ACK"
+#define PLUGINSD_KEYWORD_GAP "GAP"
+#define PLUGINSD_KEYWORD_RDATA "RDATA"
+#define PLUGINSD_KEYWORD_FILL "FILL"
+#define PLUGINSD_KEYWORD_FILLEND "FILLEND"
+// REP command with arguments 
+// on, off, ack
+typedef enum rep_arg {
+    REP_ARG_ERROR = 0,
```

Comment:
Is there a special motive to have `error` as `0`? 
Another thing, the comment above does not treat this `error`.